locales/en/translation.json

```
@@ -477,6 +473,7 @@
   "Ready!": "Ready!",
   "Recommended": "Recommended",
   "Refresh": "Refresh",
+  "Refuse": "Refuse",
```

I would not use Refuse. Reject or Decline are probably more appropriate.

```suggestion
  "Reject": "Reject",
```